packages/docs/integrate/extending-integrations.md

```diff
@@ -48,18 +48,66 @@ To register an extension, add it to the array returned from the `extensions` fun
 ```js
 module.exports = {
   integrations: {
-    <TAG NAME>: {
-      location: '@<integration-package>/server',
+    '{INTEGRATION_NAME}': {
+      location: '{SERVER_INTEGRATION}',
       configuration: {},
       extensions: (extensions) => [
         ...extensions,
         {
-          name: 'our-extension',
-          hooks: () => { /* ... */}
+          name: 'extension-name',
+          hooks: () => { /* ... */ }
         }
       ],
       customQueries: {}
     }
   }
 };
 ```
+
+## Example: Adding new API endpoints
+
+You cannot register new API endpoints directly but only through extensions to already registered integrations. Let's follow the examples above to add a new endpoint to the integration.
+
+```js
+module.exports = {
+  integrations: {
+    '{INTEGRATION_NAME}': {
+      location: '{SERVER_INTEGRATION}',
+      configuration: {},
+      extensions: (extensions) => [
+        ...extensions,
+        {
+          name: 'extension-name',
+          extendApiMethods: {
+            async customMethod(context, params) => { /* ... */ }
+          }
+        }
+      ],
+      customQueries: {}
+    }
+  }
+};
+```
```

Comment:
This wont solve the issue of users IMO. They have the similar example already. I pasted the exactly working code and I think this kind of code should be also put here as an example. Then need exact copy/paste example as they already have a docs page that explains it and they still do not know how to use it.

Reply:
The example you prepared is Vendure-specific and will not work in any other integration. That's why this section is more "generic".

Reply:
I see the purpose of having a "generic" example, but could we add some guidance to point someone to the next steps in terms of calling this endpoint from their app.

Even if it's just a sentence saying to check the specific integration guides to see how to call API endpoints for an integration.